src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -9,19 +9,71 @@ import DiscussView from './views/DiscussView';
 import ParticipantsView from './views/ParticipantsView';
 import RightPanel from '../RightPanel/RightPanel';
 import './PursuancePage.css';
+import ReactModal from 'react-modal';
 
 class PursuancePage extends Component {
-
+ constructor(){
+  super();
+  this.state={
+    modalIsOpen : false
+   }
+  this.openModal = this.openModal.bind(this);
+  this.closeModal = this.closeModal.bind(this);
+  this.handleKeydown = this.handleKeyDown.bind(this);
```

Comment:
@greg-han If we define `openModal(){ ... }` instead using the ES6 syntax `openModal = () => { ... }`, then that `openModal` function/method will use `this` from the surrounding context! And therefore binding `this` to "this" `PursuancePage` object isn't necessary using

`this.openModal = this.openModal.bind(this);`

If you look at the other React classes defined in Pursuance, you'll see that we just use arrow functions for the method definitions, and thus don't need explicit `this` binding :+1: .